components/popover_list_members/popover_list_members.jsx

```diff
@@ -124,9 +124,9 @@ export default class PopoverListMembers extends React.Component {
                             key={'popover-member-' + i}
                         >
                             <ProfilePicture
+                                className='user__picture'
```

Comment:
I don't think this prop does anything

Reply:
Well that prop exists on the portrait at the top left corner so I've added it, perhaps it will be added later so it will keep the same format. If you think it won't I can remove it, I was merely following the already existing patterns.

Reply:
It's supported by the `img` tag used there, but it doesn't do anything when you're using a `ProfilePicture` component. We should remove it since it might suggest to people using this code that this component will be affected by any CSS changes made to `user__picture`

Reply:
I see, removed.